c/include/libsbp/tracking.h

```diff
@@ -49,7 +49,7 @@ SBP_PACK_START
 #define SBP_TRACKING_STATE_DETAILED_DEP_A_SYNCHRONIZATION_STATUS_NO_SYNCHRONIZATION (0)
 #define SBP_TRACKING_STATE_DETAILED_DEP_A_SYNCHRONIZATION_STATUS_BIT_SYNCHRONIZATION (1)
 #define SBP_TRACKING_STATE_DETAILED_DEP_A_SYNCHRONIZATION_STATUS_WORD_SYNCHRONIZATION (2)
-#define SBP_TRACKING_STATE_DETAILED_DEP_A_SYNCHRONIZATION_STATUS_SUB_FRAME_SYNCHRONIZATION_MESSAGE_SYNCHRONIZATION (3)
+#define SBP_TRACKING_STATE_DETAILED_DEP_A_SYNCHRONIZATION_STATUS__SUB_FRAME_SYNCHRONIZATION__MESSAGE_SYNCHRONIZATION (3)
```

Comment:
no idea how this snuck in here.

Reply:
these were caused by the generator not stripping the yaml fields, so extra whitespace was being converted to `_`.  An extra commit has been added to prevent this occurring in future, and keep the existing ones to avoid breaking backwards compatibility.